modules/data.land/R/put.veg.module.R

```diff
@@ -49,24 +35,29 @@
     logger.severe(paste(fcn, "does not exist."))
   }
   
+  # NOTE : match_pft is now called in veg2model
```

Comment:
Ok, got it. Would it be possible to modify the check in match.species to allow DEAD/SNAG/etc to be valid unmatched species if a model hasn't set those up as being part of a PFT?